Guide/database.markdown

```diff
@@ -334,6 +334,75 @@ do
                 |> map (\(Only uuid) -> Id uuid :: Id Project)
 ```
 
+### Implementing `ToField` For Custom Types.
+
+This is a more advanced topic, and before trying to implement it, we encourage you to
+try and use the [Query Builder](https://ihp.digitallyinduced.com/Guide/querybuilder.html), as it provides a better type safety. Even the below example can be accomplished with the Query Builder.
```

Comment:
> Even the below example can be accomplished with the Query Builder.

Once we have https://github.com/digitallyinduced/ihp/issues/1867 , we should link it here.